oklog/templates/oklog.yaml

```diff
@@ -61,6 +61,31 @@ spec:
   selector:
     app: {{ template "pinpt.full_name" . }}-oklog
 
+---
```

Comment:
might want to make this opt-in/out in Values.yaml as well in case you don't need/want

Reply:
Yea, I agree with this. Let's make it similar to: https://github.com/pinpt/worker/pull/172